cdap-unit-test/src/main/java/co/cask/cdap/test/ConfigurableTestBase.java

```diff
@@ -0,0 +1,534 @@
+/*
```

Comment:
This is identical with the previous TestBase, except that init() is now called initTestBase(), has a parameter to pass in additional configuration, and is not a @BeforeClass. This will now be called from @BeforeClass TestBase.init(), such that TestBase behaves exactly as before. But we can have other test cases extend ConfigurableTestBase and pass in additional properties.